src/bootstrap/src/ferrocene/dist.rs

```diff
@@ -138,6 +141,11 @@ impl Step for SourceTarball {
             subsetter.add_file(&builder.src, &builder.src.join(item));
         }
 
+        // Generate comprehensive copyright data and include the generated files
+        for item in builder.ensure(GenerateCopyright) {
+            subsetter.add_file(&builder.src, &item);
```

Comment:
What will be the path `COPYRIGHT.html` and `COPYRIGHT-library.html` will be copied to? They should be placed at the top level, not inside the `build/` directory in the tarball (if that's the case).

Reply:
that is what happens... will adjust